lib/Basics/process-utils.cpp

```diff
@@ -225,9 +234,141 @@ static bool CreatePipes(int* pipe_server_to_child, int* pipe_child_to_server) {
 /// @brief starts external process
 ////////////////////////////////////////////////////////////////////////////////
 
-static void StartExternalProcess(ExternalProcess* external, bool usePipes,
-                                 std::vector<std::string> const& additionalEnv,
-                                 std::string const& fileForStdErr) {
+#include <spawn.h>
```

Comment:
As devel is Linux-only, we could move this include to the top of the file, to where the other includes are.